RR.UnitOfWork/Entities/ATS/Candidate.cs

```diff
@@ -78,13 +76,9 @@ public Candidate(CandidateDto candidateDto)
 
     [Column("school")] public string? School { get; set; }
 
-    [Column("degree")] public string? Degree { get; set; }
+    [Column("qualificationEndDate")] public int? QualificationEndDate { get; set; }
 
-    [Column("fieldOfStudy")] public string? FieldOfStufy { get; set; }
-
-    [Column("qualificationEndDate")] public DateOnly? QualificationEndDate { get; set; }
-
-    [Column("blacklisted")] public bool Blacklisted { get; set; }
+    [Column("blacklisted")] public BlacklistStatus Blacklisted { get; set; }
```

Comment:
Please change the wording as we discussed on Friday since it's not a boolean anymore

Reply:
updated is to BlacklistedStatus